gravitee-am-factor/gravitee-am-factor-email/src/main/java/io/gravitee/am/factor/email/provider/EmailFactorProvider.java

```diff
@@ -162,7 +162,6 @@ private Completable generateCodeAndSendEmail(FactorContext context, EmailSenderP
                     .andThen(Single.just(enrolledFactor)
                             .flatMap(ef ->  {
                                 ef.setPrimary(true);
-                                ef.setStatus(FactorStatus.ACTIVATED);
```

Comment:
We need to set the PENDING status into the MFAChallengeEndpoint line 323


`            enrolledFactor.setStatus(FactorStatus.PENDING_ACTIVATION);`